src/components/LeftSidebar/NewGroupConversation/SetContacts/ContactSelectionBubble/ContactSelectionBubble.vue

```diff
@@ -0,0 +1,106 @@
+<!--
+  - @copyright Copyright (c) 2020 Marco Ambrosini <marcoambrosini@pm.me>
+  -
+  - @author Marco Ambrosini <marcoambrosini@pm.me>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+	<div class="contact-selection-bubble">
+		<Avatar
```

Comment:
Nope :speak_no_evil:

Reply:
* This shows avatar placeholders for users instead of their proper avatar :boom: 
* This shows user avatar placeholders also for groups

Check https://github.com/nextcloud/spreed/blob/0dc9052c7694a635e307ec818a56cb9a1aa60380/src/components/RightSidebar/Participants/ParticipantsList/Participant/Participant.vue#L30-L42 instead